src/components/cars/carslist.js

```diff
@@ -13,14 +13,14 @@ export default class CarsList extends Component {
 	componentWillMount() {
 		fetch('/cars')
 			.then(res => res.json())
-			.then(cars => this.setState({ cars }));
+			.then(cars => this.props.setCars(cars));
```

Comment:
remove it
create action in redux to fetch data from server and call when store will create
so your fetch data action will start when user open your app whatever he open (/carlist.js or /about.js etc)